### PR TITLE
test(storage): add several-msg + resumable uploads to objects.insert

### PR DESCRIPTION
**Pull request by BrennaEpp**
Not all error code paths in grpc uploads are tested without uploading a larger object in the storage.objects.insert test.

Additionally, reduces the 9mb object to 3mb where possible to reduce test time.

### CODE REVIEW

**Review by cojenco — APPROVED** [review]
lgtm, thank you!

File: storage/retry_conformance_test.go [hunk 1]
```diff
@@ -392,6 +404,42 @@ var methods = map[string][]retryFunc{
 			}
 			return nil
 		},
+		// Single-shot upload that spans several message sends.
```
Comment (tritone) [inline storage/retry_conformance_test.go:407]: Note that the "message sends" part is only relevant to gRPC

**Review by tritone — APPROVED** [review]
Looks great! Can you confirm that this covers the remaining untested error paths for gRPC?

**Comment by BrennaEpp** [comment]
> Looks great! Can you confirm that this covers the remaining untested error paths for gRPC?

I tested this out and it does cover all error paths for gRPC.

**Pull request merged**